Added new article "Creating a Contact Us Form" based on Video

eshanrnh opened the pull request.

sofietoft commented on 11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md:
```diff
@@ -0,0 +1,265 @@
+## Video Tutorial
+
+{% embed url="https://www.youtube.com/watch?ab_channel=UmbracoLearningBase&v=TWLqt-jVdyQ" %}
+How to use Language Variants in Umbraco
```
This does not sound entirely correct? 🙈

sofietoft replied:
Also, the embedded video is the Language Variants for Umbraco CMS? 🤔

eshanrnh replied:
Oops.. I added a placeholder but forgot to replace it with the actual video 🙈

sofietoft commented on 11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md:
I think the tutorial needs a more thorough introduction into what the steps will take you through.
Yes, it's a contact form, but what can that be used for, etc?

sofietoft commented on 11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md:
```diff
@@ -0,0 +1,265 @@
+
+## Creating a Composition
+
+To create a Composition, follow these steps:
```
Perhaps a little more context here, as to what will be covered in this step? What is a composition, fx, and why do we need it to create a new Form?

sofietoft commented on 11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md:
```diff
@@ -0,0 +1,265 @@
+3. Select **...** next to the **Compositions** folder.
+4. Click **Composition**.  
+
+    ![Creating a Composition](images/creating-a-composition.png)
```
I think it's OK to not make the images a part of the list - the ordered list will continue under it.

sofietoft commented on 11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md:
```diff
@@ -0,0 +1,265 @@
+5. Enter a **Name** for the **Composition**. Let's call it _Title Box_.
+6. Add the following fields with the respective specifications:
+
+    | Group     | Field Name | Alias    | Data Type  |
+    |-----------|------------|----------|------------|
+    | Title Box | Title      | title    | Textstring |
+    | Title Box | Subtitle   | subtitle | Textarea   |
+
+    ![Add Composition Properties](images/composition-properties.png)
```
It's OK to not make the images/tables a part of the list - the ordered list will continue under it.

sofietoft commented on 11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md:
```diff
@@ -0,0 +1,265 @@
+
+## Configure the reCAPTCHA value
+
+To configure the reCAPTCHA value, see the [Configuration](../../developer/configuration/README.md#recaptcha-v2-field-type-configuration) article.
```
Should it be v3? 🤔

eshanrnh replied:
No, I've used v2 in the video as v2 requires user to click I'm not a robot checkbox and select the image recognition challenge. v3 is more score based and used for feedbacks

sofietoft commented on 11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md:
```diff
@@ -0,0 +1,265 @@
+8. Click **Submit**.
+9. Click **Save**.
+
+## Configure Form Settings
```
Are **all** these steps necessary?
I don't think you need to provide a stylesheet, or enter a custom label for the Submit, Next and Previous buttons fx.

eshanrnh replied:
No, all these steps are not required. In the video, I've only done 1 step ie. Select **Mark Mandatory fields** in the **Mark fields**. I can add just 1 step but it looks weird so I added the rest of the options as well. 

I can mention though that these steps are optional and not mandatory.

sofietoft reviewed (changes requested):
Overall good tutorial 💪 

I do think though, that it needs a bit more explanation here and there.
Currently, it's very much doing this, then do this, and then do this.
There is no explanation as to why each step needs to be performed.

Also, consider if it will look better than the tables and images aren't part of the lists!
GitBook recently released a feature, that allows us to continue the ordered lists under images etc.

Could it make sense to split the tutorial into high-level steps, like 1 step for creating the form, 1 step for creating the Document Type, and one step for creating the content? Not necessarily in that order, but it might make the tutorial a little easier to navigate.

Finally, how about we add a new section for the Forms space, called "Tutorials"?
We have other articles that could also go into this section eventually - it would give users a quicker way to identify tutorials!

eshanrnh commented:
Thanks for the review, @sofietoft 🙌

I've added explanation/description to each of the sections, moved the tables/images out of the lists, split the sections into high-level steps, and added a new section - Tutorials.

Let me know what you think.
